charts/operator-wandb/charts/weave-trace/templates/deployment-worker.yaml

```diff
@@ -0,0 +1,106 @@
+      annotations:
+        {{- if .Values.pod.annotations -}}
+        {{-   toYaml .Values.pod.annotations | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- if .Values.tolerations }}
+      tolerations:
+        {{- toYaml .Values.tolerations | nindent 8 }}
+      {{- end }}
+      {{- include "wandb.nodeSelector" . | nindent 6 }}
+      {{- include "wandb.priorityClassName" . | nindent 6 }}
+      {{- include "wandb.podSecurityContext" .Values.pod.securityContext | nindent 6 }}
+      containers:
+        - name: {{ include "weaveWorker.fullname" .  }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          {{- include "wandb.containerSecurityContext" .Values.container.securityContext | nindent 10 }}
+          envFrom:
+            - configMapRef:
+                name: {{ .Release.Name }}-clickhouse-configmap
+          env:
+            - name: WF_CLICKHOUSE_PASS
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "wandb.clickhouse.passwordSecret" . }}
+                  key: {{ .Values.global.clickhouse.passwordSecret.passwordKey }}
+            - name: WF_KAFKA_BROKER_HOST
+              value: "{{ .Values.bufstream.host }}"
+            - name: WF_KAFKA_BROKER_PORT
+              value: "{{ .Values.bufstream.port }}"
+            {{- include "weaveTrace.extraEnv" (dict "global" .Values.global "local" .Values) | nindent 12 }}
+            {{- include "wandb.extraEnvFrom" (dict "root" $ "local" .) | nindent 12 }}
+            - name: WANDB_INTERNAL_SERVICE_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: weave-worker-auth
+                  key: key
+            - name: WANDB_INTERNAL_SERVICE_TOKEN_SECRET_NAME
+              value: weave-worker-auth
+            - name: WANDB_BASE_URL
+              value: http://{{ .Release.Name }}-app:8080/
+            - name: WEAVE_ENABLE_ONLINE_EVAL
+              value: "true"
+            - name: WEAVE_TRACE_SERVER_BASE_URL
+              value: "{{ .Values.global.host }}/traces"
+          command:
+            - python
+          args:
+            - "-m"
+            - "src.workers.scoring_worker"
+          livenessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - ps aux | grep 'scoring_worker' | grep -v 'grep'
+            timeoutSeconds: 2
+            failureThreshold: 5
+          readinessProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - ps aux | grep 'scoring_worker' | grep -v 'grep'
+            timeoutSeconds: 2
+            failureThreshold: 5
+          startupProbe:
+            exec:
+              command:
+                - /bin/sh
+                - -c
+                - ps aux | grep 'scoring_worker' | grep -v 'grep'
+            timeoutSeconds: 2
+            failureThreshold: 5
```

Comment:
these probes do nothing, if the process in the command exits then the pod will get restarted

Reply:
yeah, if there is something to check where the process is still running then we can do that. but just checks to see if the process is running we don't need